bot.py

```
@@ -20,7 +22,83 @@
+            return await self.channel.create_thread(name=name)
+    
+    mock_message = MockMessage(interaction)
+    
+    # Use existing handler logic
+    try:
+        await handle_sum_day_command(mock_message, client.user)
+        # If no exception, the command succeeded
+        try:
+            await interaction.followup.send("✅ Daily summary has been generated!", ephemeral=True)
+        except:
+            pass  # Ignore if we can't send the followup
```

Ruff (0.11.9)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

(SIM105)